### PR TITLE
Tuning learning rate for XGBoost. Printing feature importance for XGB. Subimission #9

### PR DESCRIPTION
**Pull request by mikolajkania**

### CODE REVIEW

**Comment by mikolajkania** [comment]
Public score: 0.13698
